CSS/cards.css

```diff
@@ -0,0 +1,45 @@
+.card-child{
+    width: 100%;
+    border-radius: 0.5rem;
+    cursor: pointer;
+    border: 0.1rem solid var(--lynx-grey);
+    box-shadow: 0.3rem 0.3rem 0.1rem rgba(0,0,0,0.2);
+    position: relative;
+}
+
+.card-child:hover{
+  border: none;
+  box-shadow: none;
+}
+
+.card-secondary:hover{
+    position: absolute;
+    top: -0.8rem;
+    left: -0.8rem;
+    transition: 0.2s ease-in-out;
+    width: 18.75rem;
+    border-radius: 0.5rem;
+    border: 0.1rem solid var(--lynx-grey);
+    box-shadow: 0.3rem 0.3rem 0.1rem rgba(0,0,0,0.2);
```

Comment:
Just a suggestion, you can make a variable for box-shadow and use it in your code, it seems to be repeating